pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -224,9 +224,15 @@ func (k *Bootstrapper) init(cfg config.ClusterConfig) error {
 	}
 
 	conf := bsutil.KubeadmYamlPath
-	c := exec.Command("/bin/bash", "-c", fmt.Sprintf("%s init --config %s %s --ignore-preflight-errors=%s",
+	ctx, cancel := context.WithTimeout(context.Background(), initTimeOutMinutes*time.Second)
```

Comment:
I think `initTimeOutMinutes*time.Second` will come out to 10 seconds, when you want 10 minutes

the correct way to do this is something like:

```go
(initTimeOutMinutes * time.Minute).Seconds()
```

Reply:
oops debugging left over thanks for catching it